cpp/BUILD.bazel

```diff
@@ -123,3 +123,32 @@ cc_binary(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_library(
+    name="task_executor",
```

Comment:
Why don't use `ray_api` directly?

Reply:
Because the task_executor in the src runtime part, not visible for ray_api, so use a independent library expose it, and api_test depend the task_execute library.

Reply:
Done